diagnostic_aggregator/scripts/add_analyzers

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+
+NAME='add_analyzers'
+
+import sys
+import optparse
+from bondpy import bondpy
+from diagnostic_msgs.srv import AddDiagnostics
+import roslib; roslib.load_manifest('rosparam')
```

Comment:
This should not be necessary anymore in `catkin` Python pkgs (see [rospy tutorials - Setting Up Your PYTHONPATH](http://wiki.ros.org/rospy_tutorials/Tutorials/PythonPath)).